experimental/origin-dapp2/src/components/PageTitle.js

```diff
@@ -1,13 +1,18 @@
 import { Component } from 'react'
 
-export default class PageTitle extends Component {
+import withCreatorConfig from 'hoc/withCreatorConfig'
+
+class PageTitle extends Component {
   componentDidMount() {
-    if (!this.props.children) return 'Origin'
-    const title = this._cleanTitle(this.props.children)
-    document.title = title + ' - Origin'
+    const siteTitle = this.props.creatorConfig.title
```

Comment:
I think we can use `lodash/get` here: `const siteTitle = get(this.props, 'creatorConfig.title', 'Origin')